tests/lib/outputs/outputs_test.py

```diff
@@ -318,3 +323,57 @@ def test_extract_findings_statistics_all_fail_are_not_muted(self):
         assert stats["resources_count"] == 1
         assert stats["findings_count"] == 2
         assert not stats["all_fails_are_muted"]
+
+    def test_report_with_aws_provider(self):
```

Comment:
I'm not sure about the point of this test.

Reply:
I'll improve it to check the stdout output